README.md

```diff
@@ -130,7 +130,14 @@ includeports:
   - 80
 ```
 
-To see *only* events for these ports.
+To see *only* events for these ports. The results will still be filtered by the subnet filters. If you'd like
+the traffic from a specific port that bypasses the subnet filters, use:
+
+```yaml
+bypass_ports:
+ - 80
+ - 123
+ ```
```

Comment:
This seems like it is a bit too "lazer focused" on your immediate goal at hand, and I think a more general approach would be more flexible and less confusing to users. You could instead make the concepts defined in the current config object (filters, includeports) and make this be a list of filters/includeports. That way, you could configure internal traffic to look for ports 389 and external traffic with AMZN filtering to look at all ports. Heck, you could even have different ports/filters for each vpc :)

I know that's scope creeping your immediate goal by a good amount, but that's the "right way" to go about this IMO.